slight visual improvements on the group wrapper

its-mahdi opened the pull request:
showing groups of records again, with slight visual improvements.

xwatkins commented on src/styles/ResultsTable.css:
```diff
@@ -106,7 +106,7 @@ tbody {
 
 .group-wrapper {
-  border-bottom: 5px solid #cbcce0;
+  border-bottom: 2px solid #e0e0e0;;
```
use rem for consistency

xwatkins commented on src/ui/components/ResultsTable.jsx:
```diff
@@ -70,23 +71,35 @@ class ResultsTable extends Component {
+    return rows.map((mapping) => {
       const key = `${mapping.ensemblTranscript.enstId}_${mapping.uniprotEntry.uniprotAccession}`;
 
       // -- temp disabled for simplified table
```
Unnecessary comment

xwatkins commented on src/ui/components/ResultsTable.jsx:
```diff
@@ -70,23 +71,35 @@ class ResultsTable extends Component {
       return (
         <Link to={`${BASE_URL}/mapping/${mapping.mappingId}`} key={key} className="table-row">
           <div className="table-cell">
             {/* temporarily disabled 'isoform' flag */}
```
Same

xwatkins commented on src/ui/components/ResultsTable.jsx:
```diff
@@ -118,6 +131,22 @@ class ResultsTable extends Component {
 
+  renderBorderWrapperRows = (group) => {
+    const { wrapper } = group;
+    const {
```
None of these are used

its-mahdi commented:
Switched to 'rem' and removed the commented code. Fixed PropTypes and `key` warnings too. For 'unmapped' entries that's a different issue and we haven't agreed on any solution yet. That would be solved in its own PR.
